compiler/src/iree/compiler/Codegen/LLVMGPU/KernelConfig.cpp

```diff
@@ -86,6 +86,10 @@ static llvm::cl::opt<bool> clLLVMGPUEnablePrefetch(
     llvm::cl::desc("Enable prefetch in the vector distribute pipeline"),
     llvm::cl::init(false));
 
+static llvm::cl::opt<bool>
+    clLLVMGPUUseIgemm("iree-codegen-llvmgpu-use-igemm",
```

Comment:
nit: use similar naming to the other flags: `iree-codegen-llvmgpu-test-igemm`. We're trying to keep dev flags prefixed with `experimental` or `test` until ready to turn on by default. It helps prevent proliferation of WIP flags. We can rename it to this once on by default.